src/components/organisms/events/header/header.scss

```
@@ -0,0 +1,115 @@
+@use "../../../variables" as *;
+
+.header {
+  position: relative;
+  background-color: $black;
+  padding-left: 16px;
+  padding-right: 8px;
+
+  @media screen and (min-width: $breakpoint-xl) {
+    padding-right: 20px;
+    padding-left: 20px;
+    background-color: $black_50;
+  }
+
+  @media screen and (min-width: 1200px) {
+    .navbar-expand-xl .navbar-nav .nav-link {
+      padding-right: 20px;
+      padding-left: 20px;
+    }
+  }
+
+  .switch-img {
+    img {
+      width: 120px;
+      @media screen and (min-width: $breakpoint-xl) {
+        padding-left: 10px;
+        width: 140px;
+      }
+    }
+  }
+  .container-fluid {
+    padding: 0px;
+  }
+
+  .concept-container {
+    display: flex;
+    .humberger-text {
+      font-weight: 400;
+      font-size: $fontsize-s;
+      color: $white;
+      text-decoration: none;
+    }
+    .navbar-toggler {
+      background-color: $black;
+      margin-left: auto;
+    }
+  }
+  @media screen and (min-width: $breakpoint-xl) {
+    .container-fluid {
+      margin-left: auto;
+    }
+  }
+  .humberger-text {
+    padding-top: 15px;
+    padding-bottom: 15px;
+    @media screen and(min-width:$breakpoint-xl) {
+      padding: 20px 20px;
+      position: relative;
+      &:hover {
+        color: $gray_nobel;
+      }
+    }
+  }
+  @media screen and (min-width: $breakpoint-xl) {
+    .collaspe {
+      position: absolute;
+    }
+  }
+
+  .concept-container {
+    position: relative;
+  }
+  .dropdown-menu {
+    background-color: $black;
+    position: relative;
+    border-radius: 0%;
+
```

```suggestion
  .dropdown-menu {
    background-color: $black;
    position: relative;
    border-radius: 0%;
    
    &.collapsing {
      display: block;
    }
```
`.collapsing` は伸び縮みする間に適用されるクラスです。
現状、伸び縮みしている状態では `display: none` になっているため、ボタンを押してからメニューが開くまでタイムラグが生じてしまっていますが、上記のように修正することでなめらかに開閉するようになります。

参考：https://getbootstrap.jp/docs/5.0/components/collapse/